### ISSUE TITLE
cs-only target

### ISSUE THREAD

**Issue by pmatos** [racket/racket#2652]
Just noticed two issues with the `cs-only` target on the top-level Makefile.
 * ChezScheme is not cloned - needs to be done manually;
 * Fails to compile:
```
Use /home/pmatos/Projects/LT-racket/racket/src/cs/bootstrap/.
Use /home/pmatos/Projects/LT-racket/racket/src/build/ChezScheme/nanopass
Use /home/pmatos/Projects/LT-racket/racket/src/build/ChezScheme/s
Check /home/pmatos/Projects/LT-racket/racket/src/build/ChezScheme/boot/ta6le
Load nanopass
Apply nanopass patch
Load cmacro parts
Load enum
make-record-type: undefined;
 cannot reference an identifier before its definition
  in module: top-level
  context...:
   eval-one-top12
   /home/pmatos/Projects/LT-racket/racket/src/cs/bootstrap/make-boot.rkt:108:5: loop
   "/home/pmatos/Projects/LT-racket/racket/src/cs/bootstrap/make-boot.rkt": [running body]
   temp37_0
   for-loop
   .../more-scheme.rkt:261:28
   "/home/pmatos/installs/racket-7.2/collects/setup/main.rkt": [running body]
   temp37_0
   for-loop
   run-module-instance!125
   perform-require!78
```

Using the command line:
```
$ time make --load-average=16 CPUS=16 RACKET=/home/pmatos/installs/racket-7.2/bin/racket cs-only
```

**Comment by mflatt** [comment]
The `cs-only` target isn't meant to be public. There are many targets in the makefile that are not intended as public – although most of them have a less inviting name than `cs-only`.

I think the error you're seeing is related a bug in v7.2 fixed by 7e9d167101300144fe70f6d252c18b6242a52771. While I try to not make the build depend gratuitously on the latest version of Racket, the intent is that `RACKET=...` should work whenever the existing build is the same version (and it might also work with some other versions).

**Comment by pmatos** [comment]
OK. That makes sense. I was attempting to speed up the CS builds in CI. The idea was to have the latest stable racket3m version cached and then just build RacketCS by pointing to it but I can see it might be more trouble than it's worth. Happy for you to close the report then. Thanks!
